src/app/launcher/create-app/mission-runtime-createapp-step/mission-runtime-createapp-step.component.html

```diff
@@ -114,10 +114,10 @@ <h2 class="card-pf-title">
           <div class="card-pf card-pf--small">
             <div class="card-pf-heading">
               <h2 class="card-pf-title">
-                Choose a runtime
+                Select a runtime
```

Comment:
labels texts are not as per UXD which Brian has shared with us https://redhat.invisionapp.com/share/VGF1RF15J#/screens/286893069

Is it updated again?